client/app/Streamfinder.jsx

```diff
@@ -14,10 +14,30 @@ import MediaDetail from '../features/media/MediaDetail';
 import Account from '../features/accountPage/Account';
 import ErrorBoundary from '../features/sharedComponents/ErrorBoundary';
 import './Streamfinder.css';
-
+/* the idea:
+upon auth being valid - 
+* auth will send username and current history ids (from user schema) to StreamerFinder 
+  and state will bet set
+* Each component can use component did update for when their props
+  change.  upon this happening they can make an ajax request to server
+  to pull data needed from MovieSchema
+*/
 class Streamfinder extends React.Component {
   constructor(props) {
     super(props);
+
+    this.state = {
+      //user establish because this is prior to auth being hooked up
+      //this is under the impression auth was valid and currentId was sent
+      //to this component and updated via component did update.
+      currentId: 10138
+    }
+    this.handleSearchIdSwitch = this.handleSearchIdSwitch.bind(this)
+  }
+
+  handleSearchIdSwitch(id) {
```

Comment:
Again, it would be better to handle this in the Home feature.

Reply:
i would have to disagree - streamfinder is the entry to our app.  managing all the ids across the app will can be done efficiently with streamfinders state